src/tiles/grid/item-tile.ts

```diff
@@ -34,7 +38,11 @@ export class ItemTile extends LitElement {
             </item-image>
           </div>
           <div class="created-by truncated">
-            ${itemCreator ? html`<span>by&nbsp;${itemCreator}</span>` : nothing}
+            ${
+              this.sortParam
```

Comment:
suggestion: make this a getter so that you can make a unit test against it

Reply:
Q: when does `this.sortParam` return falsy?

Reply:
when you don't have any ?sort param in URL..